python/ray/tests/test_placement_group.py

```diff
@@ -219,16 +219,26 @@ def value(self):
         placement_group_assert_no_leak([placement_group])
 
 
-@pytest.mark.parametrize("connect_to_client", [False, True])
-def test_placement_group_spread(ray_start_cluster, connect_to_client):
-    @ray.remote(num_cpus=2)
-    class Actor(object):
-        def __init__(self):
-            self.n = 0
+def are_pairwise_unique(g):
```

Comment:
Can you move it to util? It seems to be duplicated to `test_placement_group_5`

Reply:
a bit excessive for one helper fn imo

Reply:
Then you can just import it from other test!

Reply:
ok